server/publications/roomFiles.coffee

```diff
@@ -0,0 +1,59 @@
+Meteor.publish 'roomFiles', (rid) ->
+  unless this.userId
+    return this.ready()
+
+  console.log '[publish] roomFiles'.green, rid
+
+  # list of channel messages which were created after uploading a file
+  msgQuery =
+    rid: rid
+    'file._id': { $exists: true }
+  msgOptions =
+    fields:
+      _id: 1
+      'file._id': 1
+    limit: 50
+  cursorFileMessages = RocketChat.models.Messages.find(msgQuery, msgOptions);
+  uploadedFilesMessages = cursorFileMessages.fetch()
+  unless uploadedFilesMessages
```

Comment:
As far as i know the `fetch` always returns an array, so probably this `unless` will not work, right?